autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -116,9 +116,13 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
 
   const renderMeeting = (mtg: Meeting, showSectionNum: boolean): JSX.Element => (
     <React.Fragment key={mtg.id}>
-      {showSectionNum ? createSectionHeader(mtg.section) : null }
+      {showSectionNum ? createSectionHeader(mtg.section) : null}
       <Typography className={styles.denseListItem} color="textSecondary" component="tr">
-        <td>{MeetingType[mtg.meetingType]}</td>
+        <td>
+          <Tooltip title={MeetingTypeDescription.get(mtg.meetingType)} arrow placement="right" PopperProps={{ disablePortal: true }}>
+            <span style={{ textDecorationLine: 'underline' }}>{MeetingType[mtg.meetingType]}</span>
```

Comment:
@gannonprudhomme do you think this is worth creating a CSS file for?

Reply:
Can't we move this to `SectionSelect.css`? No need to make a new file

Reply:
Done.